UI/quest_counter.csv

```diff
@@ -447,8 +447,8 @@ Basnet_GP#0,"""BP$(0)"""
 Basnet_SlctMenu_MatchStart#0,"""マッチングを開始する"""
 Basnet_SlctMenu_OrderCntRecovery#0,"""-"""
 Basnet_MessageFormTitle#0,"""バスタークエストカウンター"""
-Basnet_Proper_Order_confirm_Msg#0,"""Usted participará en una batalla de Grado: $(0)<br>Después de la victoria, obtendrás puntos de Buster.<br><yellow>※Usted se unirá a una batalla adaptada a<br>El miembro de grado más bajo del partido.<br><br>El partido se convertirá en un partido libre para<br>Los miembros que son de diferentes grados. <C><br>Al participar, esto consumirá<br>1 [Match Stamina].<br>(Match Stamina restantes: $(1))<br><br>※Usted recuperará 1 [Match Stamina] cada 5am JST. (hasta 2)<br>¿Está bien?"""
-Basnet_Support_Order_confirm_Msg#0,"""Participará en una batalla de Grado： 1～$(0).<br><br><yellow>※Si usted gana, sus Puntos Buster<br>No aumentan. Si pierde, perdera puntos.<br>Además, los objetos no caerán, no<br>Aumentan las hojas de recopilación.<br><br>[Expert Matching] no está disponible<br>para Free Matching.<c><br><br>Al obtener la victoria, podras recuperar un <br>[Match Stamina] y recibir un Match Stamina Boost.<br>（Match Stamina： $(1) Restante/hasta 3)<br><br>¿Está bien?"""
+Basnet_Proper_Order_confirm_Msg#0,"""Usted participara en una batalla de Grado: $(0)<br>Despues de la victoria, obtendras puntos de Buster.<br><yellow>※Usted se unira a una batalla adaptada a<br>El miembro de grado mas bajo del partido.<br><br>El partido se convertira en un partido libre para<br>Los miembros que son de diferentes grados. <C><br>Al participar, esto consumira<br>1 [Match Stamina].<br>(Match Stamina restantes: $(1))<br><br>※Usted recuperara 1 [Match Stamina] cada 5am JST. (hasta 2)<br>¿Esta bien?"""
```

Comment:
rejecting due to changes of `quest_counter.csv`'s `Basnet_Proper_Order_confirm_Msg#0` message